packages/api/src/db/repository.ts

```
@@ -0,0 +1,381 @@
+    coordinates_x_px: number | null;
+    coordinates_y_px: number | null;
+    is_accessible: boolean | null;
+    qr_code_payload: string | null;
+    attributes: unknown;
+    geom_x?: number | null;
+    geom_y?: number | null;
+  }): SharedNode {
+    return {
+      id: row.id,
+      name: row.name || undefined,
+      type: this.convertNodeType(row.type),
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)